base/applications/cmdutils/more/more.c

```diff
@@ -500,8 +754,102 @@ int wmain(int argc, WCHAR* argv[])
         return 1;
     }
 
+    /* Parse flags */
+    HasFiles = FALSE;
+    for (i = 1; i < argc; i++)
+    {
+        if (argv[i][0] == L'/' || argv[i][0] == L'-')
```

Comment:
We are creating a clone of a Windows command, not a linux one.
```suggestion
        if (argv[i][0] == L'/')
```
example on Windows:
```
C:\Temp>more -e
Cannot access file C:\Temp\-e
```

Reply:
![無題](https://user-images.githubusercontent.com/2107452/118381138-dc885e80-b622-11eb-9c6f-63c40fb9fbbe.png)